backend/src/rating/ranking.ts

```diff
@@ -26,10 +26,10 @@ Finally, we apply weights to each `ranking-feature` so it matches the formula pr
 */
 
 export const WEIGHTS = {
-  weight1: 60,
-  weight2: 40,
-  weight3: 100,
-  weight4: 0,
+  weight1: 70,
+  weight2: 30,
+  weight3: 60,
+  weight4: 40,
```

Comment:
The weights are different to what was discussed in the previous meeting. Please fix